.jsbeautifyrc

```diff
@@ -0,0 +1,18 @@
+{
+  "indent_size": 2,
```

Comment:
I understand that this is following the airbnb style guide, but this is just my personal opinion: Only having two spaces makes the code harder to read, especially with the callback hell that javascript can be.

Reply:
I second this...  2 spaces seems too small imo...

Reply:
You have no reason to even be dealing with that. async/await works in node 6,7 and 8. You can use promises...which eliminates callback hell. To me,  4 spaces just looks like wasted space.